src/error-handling/deriving-error-enums.md

```diff
@@ -4,11 +4,11 @@ The [thiserror](https://docs.rs/thiserror/) crate is a popular way to create an
 error enum like we did on the previous page:
 
 ```rust,editable,compile_fail
-use std::{fs, io};
+use std::{fs::{self, File}, io};
 use std::io::Read;
 use thiserror::Error;
 
-#[derive(Error, Debug)]
+#[derive(Debug, Error)]
```

Comment:
Are you sorting them alphabetically? That's not a bad idea, I'm just curious.

Also, I'll eventually do auto-formatting with `rustfmt` via something like https://dprint.dev/. So then we can have `rustfmt` do this kind of changes for us in a consistent way (though I believe sorting derives is unstable).

Reply:
Yes, I just sorted them alphabetically for consistency.